src/main/java/org/pesmypetcare/webservice/controller/petmanager/FreqTrainingRestController.java

```diff
@@ -0,0 +1,115 @@
+package org.pesmypetcare.webservice.controller.petmanager;
+
+
+import org.pesmypetcare.webservice.entity.FreqTrainingEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+import org.pesmypetcare.webservice.service.FreqTrainingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/freqTraining")
+public class FreqTrainingRestController {
+    @Autowired
+    private FreqTrainingService freqTrainingService;
+
+    /**
+     * Creates a freqTraining of the pet on the database.
+     * @param owner Username of the owner of the pet
+     * @param petName Name of the pet
+     * @param freqTrainingEntity The parameters of freqTraining
+     */
+    @PostMapping("/{owner}/{petName}/{petDate}")
+    public void createFreqTraining(@PathVariable String owner, @PathVariable String petName, @PathVariable String petDate,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 122).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5400494)